Documentation/project-docs/developer-guide.md

```diff
@@ -238,7 +242,11 @@ To run <b>only</b> the Outerloop tests, use the following command:
 msbuild <csproj_file> /t:BuildAndTest /p:WithCategories=OuterLoop
 ```
 
-_**`PlatformSpecific(TestPlatforms platforms)`:**_
+#### PlatformSpecificAttribute
+
+```cs
+PlatformSpecific(TestPlatforms platforms)
```

Comment:
and https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/TargetFrameworkMonikers.cs as appropraite